recipes/hfsm2/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "2.5.0":
+    url: "https://github.com/andrew-gresyk/HFSM2/archive/refs/tags/2.5.0.zip"
+    sha256: "e295f8bccacc9d40eb04f08414e5abe623913453bad1d2b5b70210bf8080b309"
+  "2.4.0":
```

Comment:
Do need both versions? Otherwise, I would ask adding only the latest to save the CI. In case someone asks 2.4.0, so no problem, we add it later.